app/routes/view-application.js

```diff
@@ -36,6 +36,12 @@ module.exports = {
       }
       const applicationHistory = await getApplicationHistory(request.params.reference)
 
+      const claimDataStatus = ['IN CHECK', 'REJECTED', 'READY TO PAY']
```

Comment:
could we reuse https://github.com/DEFRA/ffc-ahwr-backoffice/blob/main/app/constants/application-status.js here at all if we have the statusId available?

Reply:
Will update to use statusId